src/workflows/pyfr/components/steps/Visualization/Start/index.js

```diff
@@ -52,7 +52,7 @@ const VisualizationStart = React.createClass({
     const sessionKey = btoa(new Float64Array(3).map(Math.random)).substring(0, 96);
     const dataDir = this.props.simulation.steps.Visualization.metadata.dataDir;
     const fileName = this.props.simulation.steps.Visualization.metadata.fileName;
-    const payload = {
```

Comment:
@TristanWright @jourdain These changes showed up after I did a 'git merge master' to bring this branch up to date. Since I did not make these changes myself and they conflict with master I assume I should remove these from my branch?